templates/metube.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  createTemplate,
+  Services,
+} from "~templates-utils";
+
+export default createTemplate({
+  name: "MeTube",
+  schema: {
+    type: "object",
+    required: [
+      "projectName",
+      "serviceName",
+      "domain",
+      "downloadPath",
+    ],
+    properties: {
+      projectName: {
+        type: "string",
+        title: "Project Name",
+      },
+      serviceName: {
+        type: "string",
+        title: "App Service Name",
+        default: "metube",
+      },
+      domain: {
+        type: "string",
+        title: "Domain",
+      },
+      downloadPath: {
+        type: "string",
+        title: "Download Path",
+        default: "downloads",
+      },
+    },
+  } as const,
+  generate({
+    projectName,
+    serviceName,
+    domain,
+    downloadPath,
+  }) {
+    const services: Services = [];
+
+    services.push({
+      type: "app",
+      data: {
+        projectName,
+        serviceName: serviceName,
+        source: {
+          type: "image",
+          image: "alexta69/metube",
+        },
+        proxy: {
+          port: 8081,
+          secure: true,
+        },
+        domains: [{ name: domain }],
+        volumes: [
+          {
+            type: "volume",
+            source: downloadPath,
```

Comment:
I think we should not make this configurable. It should be "downloads" by default.

Reply:
Normally i would agree, but in this case it's pretty likely that users want to change it and at the end this is easier than explaining how to create a volume for this image afterwards.

Reply:
Makes sense. Then let's name it "Downloads Volume Name" (maybe). Or, if you want to keep it as a path, then the volume type should be `bind` not `volume`

Reply:
@Ponkhy I want to merge this PR. Can you please change "Download Path" to "Downloads Volume Name"?

Reply:
Sorry for the delay, it's renamed now.